.lintstagedrc

```diff
@@ -1,5 +1,5 @@
 {
-  "*.{js,jsx,html,css,md}": "prettier --write",
+  "*.{js,jsx,html,css}": "prettier --write",
```

Comment:
Prettifying `md` file with mdxv2 js-like comments on pre-commit leads to broken builds. Disabled for now.